packages/template-retail-react-app/app/components/_app/index.test.js

```diff
@@ -12,8 +12,19 @@ import App from '@salesforce/retail-react-app/app/components/_app/index.jsx'
 import {renderWithProviders} from '@salesforce/retail-react-app/app/utils/test-utils'
 import {DEFAULT_LOCALE} from '@salesforce/retail-react-app/app/utils/test-utils'
 import useMultiSite from '@salesforce/retail-react-app/app/hooks/use-multi-site'
-import messages from '@salesforce/retail-react-app/app/static/translations/compiled/en-GB.json'
 import mockConfig from '@salesforce/retail-react-app/config/mocks/default'
+
+let messages = {}
+try {
+    ;(async () => {
+        messages = await import(
+            '@salesforce/retail-react-app/app/static/translations/compiled/en-GB.json'
+        ).catch((err) => err)
+    })()
+} catch (err) {
+    console.log('fallback messages file not found')
+}
```

Comment:
I added some extra handling in tests for when the default translation (we use en-GB) is missing.

Reply:
What is the goal here? We expect there to always be a default locale, so shouldn't we let tests fail when it is missing? We want a smooth experience for shoppers, but we should make devs fix things the right way when they break it.

Reply:
@wjhsf I had put those there to reduce the cruft of test suites failing when a translation file is missing and left them there since the tests inside locale.test.js will still fail if the translation file is missing.

But it's also true that the translation file is very unlikely to be missing during our day to day dev unless we delete the default translation file from the monorepo so those conditions are probably unnecessary.

I'll remove them and we can re-add later if we decide we want them.